docs/concepts/usage-ideas-examples.md

```diff
@@ -116,7 +110,7 @@ The developers of the [Nix package manager](https://nixos.org/) are working on i
 
 ### Hosting software containers
 
-Netflix is on its way to use IPFS to [synchronize their Docker containers worldwide](https://blog.ipfs.tech/2020-02-14-improved-bitswap-for-container-distribution/). Since each node fetches them from the fastest peers they know, synchronization is substantially faster than regular sync methods.
+Netflix is on its way to using IPFS to [synchronize their Docker containers worldwide](https://blog.ipfs.io/2020-02-14-improved-bitswap-for-container-distribution/). Since each node fetches them from the fastest peers they know, synchronization is substantially faster than traditional synchronization methods.
```

Comment:
@DannyS03 in future edits, use .tech.

We moved blog, docs, and main website  to https://blog.ipfs.tech  :-)  and want .io to be only used for gateway at `ipfs.io/ipfs`

Reply:
Fixed in 06b83516c7f81c391800f65321e9ce259a3c6742